App.tsx

```diff
@@ -1,28 +1,37 @@
 import 'react-native-gesture-handler';
 import { StatusBar } from 'expo-status-bar';
-import React from 'react';
+import React, {useEffect} from 'react';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 
 import useCachedResources from './src/hooks/useCachedResources';
 import useColorScheme from './src/hooks/useColorScheme';
 import Navigation from './src/config/navigation';
 import { Provider } from 'react-redux';
 import { store } from './src/config/redux';
+import {ApolloClient, ApolloProvider, InMemoryCache} from "@apollo/client";
 
 export default function App() {
+
   const isLoadingComplete = useCachedResources();
   const colorScheme = useColorScheme();
 
+  const client = new ApolloClient({
+    uri: "https://graphql.anilist.co",
+    cache: new InMemoryCache(),
+  })
+
```

Comment:
It's better to move the apollo config to `src/config/apollo/index` or something, it shouldn't really be in `App.tsx`